client/src/components/Profile/StudentStatsCard.tsx

```diff
@@ -15,6 +15,7 @@ const { Text } = Typography;
 type Props = {
   data: StudentStats[];
   isEditingModeEnabled: boolean;
+  isProfileOwner: boolean | null;
```

Comment:
Why do you need null ?

Reply:
To display in a state that profile ownership wasn't cheked because the profile wasn't found. I think this is different from `false`. But perhaps this is not necessary